flink-end-to-end-tests/test-scripts/test_batch_wordcount.sh

```diff
@@ -22,4 +22,4 @@ source "$(dirname "$0")"/common.sh
 start_cluster
 
 $FLINK_DIR/bin/flink run -p 1 $FLINK_DIR/examples/batch/WordCount.jar --input $TEST_INFRA_DIR/test-data/words --output $TEST_DATA_DIR/out/wc_out
-check_result_hash "WordCount" $TEST_DATA_DIR/out/wc_out "72a690412be8928ba239c2da967328a5"
\ No newline at end of file
+check_result_hash "WordCount" $TEST_DATA_DIR/out/wc_out "72a690412be8928ba239c2da967328a5"
```

Comment:
This seems unrelated. Could you revert this?

Reply:
OKay. I will revert this soon.